feedback/feedback.go

```diff
@@ -156,12 +154,7 @@ func Print(v string) {
 
 // Warning outputs a warning message.
 func Warning(msg string) {
-	if format == Text {
-		fmt.Fprintln(feedbackErr, msg)
-	} else {
-		bufferWarnings = append(bufferWarnings, msg)
-	}
-	logrus.Warning(msg)
+	fmt.Fprintln(feedbackErr, msg)
```

Comment:
I would re-add this anyway



```suggestion
	if format == Text {
		fmt.Fprintln(feedbackErr, msg)
	} else {
		bufferWarnings = append(bufferWarnings, msg)
	}
```